doc/nrf/protocols/lte/index.rst

```diff
@@ -0,0 +1,217 @@
+| Bandwidth                        | 1.4 MHz                          | 200 kHz                          |
++----------------------------------+----------------------------------+----------------------------------+
+| Antenna Techniques               | Single antenna                   |  Single antenna                  |
++----------------------------------+----------------------------------+----------------------------------+
+| OFDM subcarriers                 |  OFDM subcarriers (72)           | OFDM subcarriers (12)            |
++----------------------------------+----------------------------------+----------------------------------+
+| Duplexing                        | Can use half-duplex              | Must use half-duplex             |
++----------------------------------+----------------------------------+----------------------------------+
+| Modulation in :term:`Uplink (UL)`| QPSK or 16QAM in both UL and DL  | BPSK or QPSK in UL depending on  |
+| and :term:`Downlink (DL)`        | depending on signal quality      | signal quality.                  |
+|                                  |                                  | Only QPSK in DL                  |
++----------------------------------+----------------------------------+----------------------------------+
+
+Network support
+================
+
+A factor governing the selection of the access technology is the MNO support in the areas where you wish to deploy your product.
+Most cellular operators support both LTE-M and NB-IoT, but there are geographic locations where only one of these is supported.
+See `Mobile IoT Deployment Map`_ for more information.
+
+Certifications
+==============
+
+Some MNOs, such as Verizon and Vodafone, have their own certification requirements.
+But many operators only require GCF (Global Certification Forum) and PTCRB certifications that devices work as per standard, and regulatory certifications.
+
+For more information about certification related to the nRF91 Series SiP, refer to the following pages:
+
+* `nRF91 Series certifications`_
+* `Certifying a Cellular IoT Device`_
+
+SIM card support
+================
+
+SIMs that are used with the nRF91 Series devices must support LTE-M, NB-IoT, or both.
+The iBasis SIM is bundled with the nRF9160 DK and Thingy:91, and Onomondo SIM is bundled with the nRF9161 DK.
+Check the `iBasis IoT network coverage`_, `Onomondo LTE-M coverage`_, or `Onomondo NB-IoT coverage`_ pages to see the network coverage for different countries, depending on the SIM card you are using.
+
+Software SIM
+------------
+
+The nRF91 Series supports software SIM, which allows the usage of software SIM-based solutions to reduce energy consumption associated with physical SIMs.
+The following are some of the key advantages of using software SIM:
+
+* Cost savings and simplified design by excluding the SIM hardware layer.
+* Power savings, especially in idle mode using eDRX.
+
```

Comment:
Should probably say the fact  here that traditional carriers (not MVNOs like iBasis or Onomondo and alike) will most likely not allow their profiles be provisioned/uploaded into SoftSIMs. One need to use profiles coming from the SoftSIM provides (such as Onomondo).

Reply:
Updated. Please check.